src/webots/nodes/WbTrack.cpp

```diff
@@ -908,8 +885,7 @@ void WbTrack::exportAnimatedGeometriesMesh(WbVrmlWriter &writer) const {
 
   if (writer.isX3d()) {
     writer << "<Transform ";
-    writer << "translation='" << position << "' ";
-    writer << "rotation='" << rotation << "'>";
+    writer << " type='animatedgeometry'>";
```

Comment:
There are two consecutive spaces, here, it would be simpler to write a single line instead, like:
```c++
    write << "<Transform type='animatedgeometry'>";
```